{"uuid": "14b0b370-bcb2-4b42-87f2-9294a9771cdb", "children": ["20733d99-07b4-4479-936b-313f0d04be72"], "befores": [{"name": "setup_driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\conftest.py\", line 56, in setup_driver\n    web_driver = webdriver.Chrome(\n                 ^^^^^^^^^^^^^^^^^\n"}, "start": 1712759234348, "stop": 1712759234348}], "start": 1712759234348, "stop": 1712759234357}